client/components/phone-input/country-flag.jsx

```diff
@@ -48,15 +48,12 @@ export default class extends React.Component {
 
 		if ( this.props.countryCode ) {
 			if ( ! this.state.error ) {
-				return (
-					<img
-						onLoad={ this.handleImageLoad }
-						onError={ this.handleImageError }
-						src={ `/calypso/images/flags/${ this.props.countryCode }.svg` }
-						className="phone-input__flag-icon"
-						style={ style }
-					/>
-				);
+				const svg = require( 'svg-inline-loader!' +
+					'../../../node_modules/flag-icon-css/flags/4x3/' +
+					this.props.countryCode +
+					'.svg' );
+				console.log( svg );
+				return <div dangerouslySetInnerHTML={ { __html: svg } } />;
```

Comment:
check out https://www.npmjs.com/package/svg-inline-react

I think that `svg-inline-loader` spits out a data-uri and the browser might even render that when console log'd

you can also try doing `console.log( JSON.stringify( svg ) )` to confirm

Reply:
alternatively you should be able to do this if indeed we have a data-uri

```js
<img src={ svg } />
```